skorch/tests/callbacks/test_lr_scheduler.py

```diff
@@ -241,6 +241,32 @@ def test_lr_scheduler_record_batch_step(self, classifier_module, classifier_data
         )
         assert np.all(net.history[-1, 'batches', :, 'event_lr'] == new_lrs)
 
+    @pytest.mark.parametrize('policy, kwargs', [
+        (TorchCyclicLR, {'base_lr': 123, 'max_lr': 999})
+    ])
+    def test_cyclic_lr_with_epoch_step_warning(self,
+                                               classifier_module,
+                                               classifier_data,
+                                               policy,
+                                               kwargs):
+        with pytest.warns(None) as record:
+            scheduler = LRScheduler(
+                policy, **kwargs)
```

Comment:
We do not need to parametrize

```suggestion
                TorchCyclicLR, base_lr=123, max_lr=999)
```